src/screens/Authentication/Verify.tsx

```diff
@@ -34,26 +35,9 @@ interface VerifyProps {
 }
 
 export const Verify = ({ onBack, userCode, verificationUrl }: VerifyProps) => {
-  const dialog = React.useRef<Window>();
-
-  // Close the dialog window when the screen gets unmounted.
-  React.useEffect(() => () => dialog.current?.close(), []);
-
-  const openChromatic = () => {
-    const width = 800;
-    const height = 800;
-    const usePopup = window.innerWidth > width && window.innerHeight > height;
-
-    if (usePopup) {
-      const left = (window.innerWidth - width) / 2 + window.screenLeft;
-      const top = (window.innerHeight - height) / 2 + window.screenTop;
-      const options = `scrollbars=yes,width=${width},height=${height},top=${top},left=${left}`;
-      dialog.current = window.open(verificationUrl, "oauth-dialog", options);
-      if (window.focus) dialog.current.focus();
-    } else {
-      dialog.current = window.open(verificationUrl, "_blank");
-    }
-  };
+  const openChromatic = useChromaticDialog({
+    login: () => openChromatic(verificationUrl),
+  });
```

Comment:
Passing a plain object will cause the event listener in `useChromaticDialog` to be rebound on every render of `Verify`, since it's a fresh new `handlers` object each time. Probably better to wrap it in a `useMemo` first.

Reply:
Good call.